### ISSUE TITLE
Building & Using locally built Rust Matrix SDK settings are faulty

### ISSUE THREAD

**Issue by theIDinside** [vector-im/element-x-android#282]
### Steps to reproduce

I followed the developing onboarding for building rust-matrix-sdk. Those steps were correct

### Outcome

I could verify that the `.aar` file had the new symbols that I had introduced in that repo. But I couldn't get Android Studio to find these symbols, it just kept claiming unresolved references. Opening the `matrix-rust-sdk.aar` file showed my functions that I had exposed.

I commented out the `implementation(libs.matrix.sdk)` - however, the onboarding docs confusingly say this

```groovy
dependencies {
    api(projects.libraries.rustsdk) // <- comment this line
    // api(libs.matrix.sdk) // <- uncomment this line
}
```

Which would make one think that one should _comment_ the line with `rustsdk` when in fact - one should _uncomment_ that line and comment the other line (following the onboarding would mean you're just keeping the default settings).

However, the problem was still not fixed, after having commented out `api(libs.matrix.sdk)` and _uncommenting_  `api(projects.libraries.rustsdk)`. Android studio could still not find the new symbols I had introduced - and Android Studio complained that it could not find `api(projects.libraries.rustsdk)` either. After many attempts and hours, it seems as though the file inside the `/libraries/rustsdk/build.gradle` is _not_ recognized by the build system (or it's current settings) - and I looked at all other files and they had the `.kts` extension after it - so I added it - and then it worked.

So in order to build with a locally built rust sdk, I needed to add that extension name to the `build.gradle` file inside `/libraries/rustsdk`.

But as I don't have all that much experience with Kotlin or Android development, I figured I'd make an issue about this, before submitting anything. Because I don't know why that solved the issue.

But I'm pretty confident it did solve the issue, because once I did, `Android Studio` immediately parsed the file and complained that `artifacts.add("default", file('matrix-rust-sdk.aar'))` had an invalid character; i.e. it had to have `"` around the name and not `'`

After updating that as well, it now builds.

### Will you send logs?

Yes

### Are you willing to provide a PR?

Yes

**Comment by jmartinesp** [comment]
Could you confirm if this was fixed by https://github.com/vector-im/element-x-android/pull/287?

**Comment by theIDinside** [comment]
This was fixed by #287 as suggest by me in this issue.
